extension/sidebar-pane.js

```diff
@@ -38,13 +43,28 @@ async function _updateSubtree(selectedNode) {
   progressEl.textContent = "Getting web compatibility issues";
   issueListEl.innerHTML = "";
   _recursiveNodesIssuesRendering(0, nodesInSubtree, false, issueListEl, []).then(() => {
+    if (!_isCurrentProcess(issueListEl)) {
+      return;
+    }
+
     subtreeEl.classList.remove("processing");
     if (!issueListEl.querySelector("li")) {
       _renderNoIssue(issueListEl);
     }
   });
 }
 
+function _getNewULElement(selector) {
+  const oldElement = document.querySelector(selector);
+  const newElement = document.createElement("ul");
+  oldElement.parentElement.replaceChild(newElement, oldElement);
```

Comment:
Isn't the `oldElement` going to leak with this?
It looks to me like it is going to become detached, but still referenced by some JS code that will keep on appending elements to it, and we won't be able to ever delete it.
We should call `remove()` at some stage to make sure it gets cleaned up, right?